library/Icinga/Cli/Command.php

```diff
@@ -94,7 +95,7 @@ private function getMainConfig($file = null)
             return $this->config;
         } else {
             if (! array_key_exists($file, $this->configs)) {
-                $this->configs[$file] = Config::module($module, $file);
+                $this->configs[$file] = Config::module($this->moduleName, $file);
```

Comment:
I know, you didn't touch this, but `getMainConfig` is not about module configs. You've faced this issue probably because the namespace of your CLI command is/was wrong and as such `Config` decided to call `getMainConfig`. Which is indeed bugged.

The actual required fix would probably be:
```suggestion
                $this->configs[$file] = Config::app($file);
```